zone/client_packet.cpp

```diff
@@ -3038,8 +3042,22 @@ void Client::Handle_OP_AssistGroup(const EQApplicationPacket *app)
 		LogDebug("Size mismatch in OP_AssistGroup expected [{}] got [{}]", sizeof(EntityId_Struct), app->size);
 		return;
 	}
-	QueuePacket(app);
-	return;
+
+	EntityId_Struct* eid = (EntityId_Struct*)app->pBuffer;
+	Entity* entity = entity_list.GetID(eid->entity_id);
+
+	EQApplicationPacket* outapp = new EQApplicationPacket(OP_Assist, sizeof(EntityId_Struct));
+	eid = (EntityId_Struct*)outapp->pBuffer;
+	if (entity && entity->IsMob()) {
```

Comment:
Didn't you want to zero out / initialize `entity_id` here?

```
eid->entity_id = 0;
```

Reply:
I took another look and made a few updates.  It now only sends the packet if required.  Also I noted that I was note deleting outapp properly, so fixed that.  If you could take another look, that would be most appreciated.